apps/oracles/gecko-terminal/src/lib.rs

```diff
@@ -63,10 +80,42 @@ async fn oracle_request(settings: Settings) -> Result<Payload> {
         let string = String::from_utf8(body)?;
         let value: GeckoTerminalResponce = serde_json::from_str(&string)
             .context("Couldn't parse Gecko terminal response properly")?;
-        let price = if config.arguments.reverse {
-            value.reserve_price_in_usd()
+        Ok(value)
+    }
+}
+
+#[oracle_component]
+async fn oracle_request(settings: Settings) -> Result<Payload> {
+    println!("Starting oracle component - Gecko Terminal");
+    let mut payload: Payload = Payload::new();
+    let gecko_config = get_resources_from_settings(&settings)?;
+    for config in gecko_config {
+        let mut total = 0.0f64;
```

Comment:
```suggestion
        let mut total: f64 = 0.0f64;
```

Just for consistency